azure-webapp-maven-plugin/src/test/java/com/microsoft/azure/maven/webapp/utils/TestUtils.java

```diff
@@ -9,26 +9,78 @@
 import java.io.IOException;
 
 public class TestUtils {
+    private static final String clientId = System.getenv("CLIENT_ID");
+    private static final String tenantId = System.getenv("TENANT_ID");
+    private static final String key = System.getenv("KEY");
+
+    private static final boolean isWindows = System.getProperty("os.name").contains("Windows");
+
+    private static void azureLogin() throws IOException, InterruptedException {
+        final String azLoginTemplate = "az login --service-principal -u %s -p %s --tenant %s";
+        Runtime.getRuntime().exec(formatCommand(azLoginTemplate, new String[]{clientId, key, tenantId})).waitFor();
```

Comment:
We have a lot of `Runtime.getRuntime().exec(xxx).waitFor()` here. How about extract a method to wrap this?

Reply:
good idea